mobile/cmp/mask/Mask.js

```diff
@@ -60,7 +60,7 @@ export class Mask extends Component {
                 items: [
                     showSpinner ? progressCircular({indeterminate: true}) : null,
                     showSpinner ? vspacer(10) : null,
-                    message ? box({className: 'xh-mask-text', item: message}) : null
+                    message ? div({className: 'xh-mask-text', item: message}) : null
```

Comment:
No, no particular reason - except we're not using LayoutSupport. There's no problem with using box here, just a question of preference :)

Reply:
My 2 rials:
I was thinking about it slightly differently from you, where `div` is the "base unit" and we should only switch to `box` when actually using `layoutSupport` props. I think its important because `box` _isn't_ a drop-in replacement for `div` - it changes how a bunch of props are handled, and while div > box changes shouldn't cause issues, box > div changes can break layouts. We shouldn't think of them as interchangeable, since they serve different purposes.

Reply:
Yes I agree with you there - no reason to reach for box if you truly just need a div and don't plan on using any layout props (things like padding included).  

Also (to state the obvious) nothing wrong with a div + CSS class to style however required.